networkx/algorithms/tests/test_clique.py

```diff
@@ -233,6 +233,17 @@ def test_directed(self):
         with pytest.raises(nx.NetworkXNotImplemented):
             next(nx.find_cliques(nx.DiGraph()))
 
+    def test_find_cliques_trivial(self):
+        G = nx.Graph()
+        assert sorted(nx.find_cliques(G)) == []
+        assert sorted(nx.find_cliques_recursive(G)) == []
+
+    def test_make_max_clique_graph_2(self):
+        "Test the create_using parameter"
+        G = nx.Graph()
+        G.add_edges_from([(1, 2), (3, 1), (4, 1), (5, 6)])
+        assert sorted(nx.make_max_clique_graph(G, nx.Graph)) == [0, 1, 2, 3]
```

Comment:
This check isn't very specific - it basically just checks that there are 4 nodes in the max_clique_graph, but doesn't test anything else about it. What do we expect the output graph of `max_clique_graph` to look like in this case?

Reply:
True, I will modify it so we can check that the edges between the cliques are ok too. With this modification, we know that the cliques are found and the edges between the cliques that share nodes are added. In this case, we have 4 cliques and 3 of them share nodes so these should form a connected component in the maximal clique graph.